src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -129,13 +144,30 @@ export default function SettingsLMSTab(enterpriseId) {
         Enabling a learning management system for your edX account allows quick
         access to the catalog
       </p>
+      {displayNeedsSSOAlert && (
+        <Alert
+          className="mr-6 sso-alert-modal-margin"
+          variant="danger"
+          icon={Info}
+          actions={[
+            <Link to={`/${enterpriseSlug}/admin/settings/sso`}>
+              <Button>Configure SSO</Button>
+            </Link>,
+          ]}
+        >
+          <Alert.Heading>No SSO configured</Alert.Heading>
+          <p>
+            At least one Single Sign On configuration is needed to create an LMS configuration
```

Comment:
Did we want to block LMS creation on SSO? I think we simply want to warn the user that they need to create an SSO config (we can confirm with Alena but I am pretty sure we wanted to not disallow LMS config creation, just that we want to let users know SSO is also going to be needed?) Sorry if any confusions